themes-default/slim/src/components/display-show.vue

```diff
@@ -447,17 +447,17 @@ export default {
         // @TODO: OMG: This is just a basic json, in future it should be based on the CRUD route.
         // Get the season exceptions and the xem season mappings.
         getSeasonSceneExceptions() {
-            const indexerName = document.querySelector('#indexer-name').value;
-            const seriesId = document.querySelector('#series-id').value;
-            if (!indexerName || !seriesId) {
+            const { indexer, id } = this;
+
+            if (!indexer || !id) {
                 console.warn('Unable to get season scene exceptions: Unknown series identifier');
                 return;
             }
-            const params = {
-                indexername: indexerName,
-                seriesid: seriesId
-            };
-            apiRoute.get('home/getSeasonSceneExceptions', { params }).then(response => {
+
+            apiRoute.get('home/getSeasonSceneExceptions', {
```

Comment:
You must put the parameters in an object like `{ params: { a: 1, b: 2 } }` (or like it was before `{ params }`), because the second argument is `config`.
[`axios.post`, `axios.put` and `axios.patch` support what you're after, but the other methods don't](https://github.com/axios/axios/tree/v0.18.0#request-method-aliases).